systests/cdi/base/src/main/java/org/apache/cxf/systests/cdi/base/OpenTracingProducer.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.cxf.systests.cdi.base;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+import org.apache.cxf.tracing.opentracing.jaxrs.OpenTracingFeature;
+
+@ApplicationScoped
+public class OpenTracingProducer {
```

Comment:
I couldn't get automatic registration to work.  Not sure why, but I went with the hint from Sergey and added an explicit producer to make sure it was a CDI bean.

Reply:
I will take a look, should be working (in general but we haven't tested with CDI)

Reply:
Adding `bean.xml` to `opentracing` module fixed the issue right away, the `OpenTracingFeature` has been discovered. As a side note, could you please move these testcases to `systest-tracing` project, it would be good to keep the CDI tests clean from external integration. Thank you.